nengo/dists.py

```diff
@@ -99,9 +102,9 @@ class Uniform(Distribution):
         If true, sample from a uniform distribution of integers. In this case,
         low and high should be integers.
     """
-    low = NumberParam()
-    high = NumberParam()
-    integer = BoolParam()
+    low = NumberParam('low')
```

Comment:
This `name = Class('name')` theme makes me kind of :grimacing:.  It's exactly the kind of thing we were trying to avoid when we made `label` optional for other Nengo objects.  I can see why we want to make names mandatory here though, since `config.set_param(Param(...))` needs to have a name.  I don't have a great solution, just complaining.

The only thing that comes to mind is to go back to the old syntax of `set_param(name, parameter)`, (so we're sure to have a name), and then have `set_param` do `parameter.label = name`.  It's a bit awkward though, because it's not obvious what to do if they already set a label for the parameter.